scripts/deploy-explorer-linux.sh

```diff
@@ -0,0 +1,15 @@
+HASH_COMMIT=$1
+
+ssh staging@35.156.156.28
```

Comment:
I'd rather see this being invoked within iohk-nixops repo for two reasons:

- not to expose IP of deployer box
- this will be used for different features as well so folder path won't match